project/Dependencies.scala

```diff
@@ -38,7 +38,7 @@ object Dependencies {
     // TODO: can these move to sbt's dependencyOverrides?
     "io.netty"                       % "netty-handler"       % nettyV, // netty is needed by the Elasticsearch client at runtime
     "org.apache.lucene"              % "lucene-queryparser"  % "6.6.6", // pin to this version; it's the latest compatible with our elasticsearch client
-    "com.google.guava"               % "guava"               % "33.4.5-jre",
+    "com.google.guava"               % "guava"               % "33.4.0-jre",
```

Comment:
Does this project use `guava`? I don't see it imported anywhere and `sbt assembly` works with this line commented out. But maybe it's needed for some kind of runtime dependency, I didn't try running it.

Reply:
I will investigate that when reverting the revert!